src/Authorization/Authorization.router.js

```diff
@@ -0,0 +1,118 @@
+import { server, env } from 'decentraland-commons'
+
+import { Approval } from '../Approval'
+import { unsafeParseInt } from '../lib'
+import { Parcel, Estate } from '../Asset'
+import { ASSET_TYPES } from '../shared/asset'
+
+export class AuthorizationRouter {
+  constructor(app) {
+    this.app = app
+  }
+
+  mount() {
+    /**
+     * Returns the bids for a parcel
+     * @param  {string} x
+     * @param  {string} y
+     * @param  {string} [address] - address to get authorizations
+     * @return {<id, address, isOwner, isApprovedForAll, isOwner, isOperator, isUpdateOperator>}
+     */
+    this.app.get(
+      '/parcels/:x/:y/:address/authorizations',
+      server.handleRequest(this.getParcelAuthorizations.bind(this))
+    )
+
+    /**
+     * Returns the bids for a parcel
+     * @param  {string} id
+     * @param  {string} [address] - address to get authorizations
+     * @return {<id, address, isOwner, isApprovedForAll, isOwner, isOperator, isUpdateOperator>}
+     */
+    this.app.get(
+      '/estates/:id/:address/authorizations',
+      server.handleRequest(this.getEstateAuthorizations.bind(this))
+    )
+  }
+
+  async getAuthorizations(asset, assetType, address) {
+    let tokenAddress
+
+    switch (assetType) {
+      case ASSET_TYPES.parcel: {
+        tokenAddress = env.get('LAND_REGISTRY_CONTRACT_ADDRESS')
+        break
+      }
+      case ASSET_TYPES.estate: {
+        tokenAddress = env.get('ESTATE_REGISTRY_CONTRACT_ADDRESS')
+        break
+      }
+      default:
+        throw new Error(`The assetType ${assetType} is invalid`)
+    }
```

Comment:
Two things about this one:

- I think we should use contractAddresses here instead of `env.get` as we do in the rest of the codebase...except Estate.modal, but we need to change that one
- *Not important*: You think it's good idea if we add a helper for this? I _think_ we needed it elsewhere

Reply:
The helper will work.  We should need where we use `contractAddresses` with a switch by `ASSET_TYPE` ([issue created](https://github.com/decentraland/marketplace/issues/875))

Reply:
I should revert it because when we start the server we are not connecting to the eth which is need for `ethereum.js` to work

Reply:
Ah that's a shame. Maybe we can call `loadContracts` from the server instead (it doesn't connect to a node). We'll leave it to the other issue